### PR TITLE
Jb57471

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: sdk-setup/src/mb2 [hunk 1]
```diff
@@ -2343,8 +2343,8 @@ sign_rpms__prepare() {
 }
 
 sign_rpms() {
-    [[ -e $STATEDIR/spec ]] || fatal "No previous build found"
-    local rpms=$(find_rpms $all_opt "$OPT_OUTPUTDIR" "$OPT_PKGDIR" "$STATEDIR/spec")
+    [[ -e $TMP_SPEC ]] || fatal "No previous build found"
```
Comment (vigejolla) [inline sdk-setup/src/mb2:2376]: Just wondering if we should pass the spec file as argument for these functions as well

File: sdk-setup/src/mb2 [hunk 2]
```diff
@@ -1512,6 +1512,26 @@ get_spec_tag() {
         --queryformat="$query" "$spec"
 }
 
+# Do a simple, one level expansion of %include directions. Failure to process
+# an %include directive is not considered fatal and as such is copied
+# unchanged to the output.
+flattern_spec() {
```
Comment (vigejolla) [inline sdk-setup/src/mb2:1518]: flatten?

File: sdk-setup/src/mb2 (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/mb2]: yup, fixed, thanks

File: sdk-setup/src/mb2 (hunk 2 above)
Comment (martyone) [reply sdk-setup/src/mb2]: Fixed

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**